GVFS/GVFS.Service/RepoRegistry.cs

```diff
@@ -162,31 +168,28 @@ public bool TryRemoveRepo(string repoRoot, out string errorMessage)
             return false;
         }
 
-        public void AutoMountRepos(int sessionId)
+        public void AutoMountRepos(string userId, int sessionId)
         {
             using (ITracer activity = this.tracer.StartActivity("AutoMount", EventLevel.Informational))
             {
-                using (GVFSMountProcess process = new GVFSMountProcess(activity, sessionId))
+                List<RepoRegistration> activeRepos = this.GetActiveReposForUser(userId);
+                if (activeRepos.Count == 0)
                 {
-                    List<RepoRegistration> activeRepos = this.GetActiveReposForUser(process.GetUserId());
-                    if (activeRepos.Count == 0)
-                    {
-                        return;
-                    }
+                    return;
+                }
 
-                    this.SendNotification(sessionId, "GVFS AutoMount", "Attempting to mount {0} GVFS repo(s)", activeRepos.Count);
+                this.SendNotification(sessionId, "GVFS AutoMount", "Attempting to mount {0} GVFS repo(s)", activeRepos.Count);
 
-                    foreach (RepoRegistration repo in activeRepos)
+                foreach (RepoRegistration repo in activeRepos)
+                {
+                    // TODO #1043088: We need to respect the elevation level of the original mount

```

Comment:
Is this still relevant? Should this be updated with a Github work item?

Reply:
it is still relevant on Windows. Updated with Github [issue #](https://github.com/Microsoft/VFSForGit/issues/1089)